pyvista/plotting/charts.py

```diff
@@ -3049,13 +3049,13 @@ def _parse_format(cls, fmt):
         # Note: All colors, marker styles and line styles are sorted in decreasing order of length to be able to find
         # the largest match first (e.g. find 'darkred' and '--' first instead of 'red' and '-')
         colors = sorted(
-            itertools.chain(pyvista.hexcolors.keys(), pyvista.color_char_to_word.keys()),
+            itertools.chain(hexcolors.keys(), color_synonyms.keys()),
             key=len,
             reverse=True,
         )
         marker_styles = sorted(ScatterPlot2D.MARKER_STYLES.keys(), key=len, reverse=True)
         line_styles = sorted(Pen.LINE_STYLES.keys(), key=len, reverse=True)
-        hex_pattern = "#[A-Fa-f0-9]{6}"
+        hex_pattern = "(#|0x)[A-Fa-f0-9]{6}([A-Fa-f0-9]{2})?"  # Match RGB(A) hex string
```

Comment:
No it's optional for sure when creating a `Color` instance. For this `parse_format` method however, a prefix is required as otherwise there could be a conflict with marker style `'d'`: Suppose the input format string is `'d00000d'`, we would be unable to determine whether the color is `#00000d` or `#d00000` (combined with marker style `'d'`. If a prefix is specified we know for sure which case it is.

Reply:
OK, perfect, as long as it's consistent I'm happy :)